packages/manifests/polywrap/formats/polywrap.docs/0.1.0.json

```diff
@@ -0,0 +1,104 @@
+{
+  "id": "DocsManifest",
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["format", "title"],
+  "properties": {
+    "format": {
+      "type": "string",
+      "description": "Polywrap Docs manifest format version.",
+      "enum": ["0.1.0"]
+    },
+    "title": {
+      "type": "string",
+      "description": "The title of the docs."
+    },
+    "logo": {
+      "type": "string",
+      "description": "Path to the wrap's logo."
+    },
+    "website": {
+      "type": "string",
+      "description": "Website related to the wrap."
+    },
+    "github": {
+      "type": "string",
+      "description": "Github repository link."
+    },
+    "pages": {
+      "type": "object",
+      "description": "Object containing sidebar pages where the object's keys are page slugs.",
+      "additionalProperties": false,
+      "patternProperties": {
+        "^([a-z\\-])+$": {
+          "type": "object",
+          "required": ["title", "path"],
+          "additionalProperties": false,
+          "properties": {
+            "title": {
+              "type": "string",
+              "description": "The title of the page."
+            },
+            "path": {
+              "type": "string",
+              "description": "Path to the page"
+            }
+          }
+        }
+      }
+    },
+    "homePage": {
+      "type": "string",
+      "description": "The slug of the home page.",
+      "pattern": "^([a-z\\-])+$"
+    },
+    "examples": {
+      "type": "object",
+      "description": "Object containing examples where the the object's keys are example slugs.",
+      "additionalProperties": false,
+      "patternProperties": {
+        "^([a-z\\-])+$": {
+          "type": "object",
+          "required": ["title"],
+          "additionalProperties": false,
+          "properties": {
+            "title": {
+              "type": "string",
+              "description": "The title of the example."
+            },
+            "steps": {
+              "type": "array",
+              "description": "Wrap invocations as an array of example steps.",
+              "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "required": ["uri", "method"],
+                "properties": {
+                  "description": {
+                    "type": "string",
+                    "description": "A description of the step rendered above the invocation code."
+                  },
+                  "uri": {
+                    "description": "URI of the wrap being invoked.",
```

Comment:
I would make this URI field optional. If it isn't specified it is assumed it is a self-invocation. If it is it will be used.

No need to introduce a special "$$WRAP_URI" or "$$THIS_URI", just leave it empty.

Reply:
One thing we can't do without `$$WRAP_URI` is add it to the `description` of the step, but I don't think it's important, so off it goes.

Reply:
Nice :D